src/main/scala/temple/generate/CRUD.scala

```diff
@@ -1,13 +1,6 @@
 package temple.generate
 
-abstract class CRUD(val verb: String)
-
-object CRUD {
-  case object ReadAll extends CRUD("List")
-  case object Create  extends CRUD("Create")
-  case object Read    extends CRUD("Read")
-  case object Update  extends CRUD("Update")
-  case object Delete  extends CRUD("Delete")
-
-  val values = Seq(ReadAll, Create, Read, Update, Delete)
+object CRUD extends Enumeration {
+  type CRUD = Value
```

Comment:
Where for art tho value defined?

Reply:
@jaylees14 it’s a field of the Enumeration class: https://www.scala-lang.org/api/current/scala/Enumeration.html